[MI-3417] Added the documentation for slash commands (#246)
mattermost/mattermost-plugin-msteams-sync: README.md

    - Click on that link. If it asks for login, enter the Microsoft credentials for the dummy account created following [these steps](./docs/azure_setup.md#step-2-create-a-user-account-to-act-as-a-bot).
    - Refer [here](./docs/azure_setup.md#step-2-create-a-user-account-to-act-as-a-bot) for more details.

## Slash commands

- `/msteams-sync connect` :- This command is used to connect your Mattermost account to MS Teams account.
- `/msteams-sync disconnect` :- This command is used to disconnect your Mattermost account from MS Teams account.
- `/msteams-sync connect-bot` :- This command is used to connect the bot account in Mattermost to an account in MS Teams and it can only be run by system admins.
- `/msteams-sync disconnect-bot` :- This command is used to disconnect the bot account in Mattermost from the MS Teams account and it can only be run by system admins.

> - `/msteams-sync link` :- This command is used to link the currently active Mattermost channel with an MS Teams channel and it can only be run by users who have roles channel admins and above. To run this command, you must your Mattermost account connected with MS Teams. This command takes two arguments - MS Teams team ID and channel ID. You can get both these values using the slash command autocomplete that shows up after you type or select the link command.

Reviewer: ```suggestion
- `/msteams-sync link` :- This command is used to link the currently active Mattermost channel with an MS Teams channel and it can only be run by users who have roles of channel admins and above. To run this command, you must have your Mattermost account connected with MS Teams. This command takes two arguments - MS Teams team ID and channel ID. You can get both these values using the slash command autocomplete that shows up after you type or select the link command.
```

> - `/msteams-sync link` :- This command is used to link the currently active Mattermost channel with an MS Teams channel and it can only be run by users who have roles channel admins and above. To run this command, you must have your Mattermost account connected with MS Teams. This command takes two arguments - MS Teams team ID and channel ID. You can get both these values using the slash command autocomplete that shows up after you type or select the link command.

Reviewer 2: ```suggestion
- `/msteams-sync link` :- This command is used to link the currently active Mattermost channel with an MS Teams channel and it can only be run by users who are channel admins and above. To run this command, you must have your Mattermost account connected with MS Teams. This command takes two arguments - MS Teams team ID and channel ID which you can get from command autocomplete.
```

> - `/msteams-sync unlink` :- This command is used to unlink the currently active Mattermost channel with the MS Teams channel and it can only be run by users who have roles channel admins and above. To run this command, you don't need to have your Mattermost account connected to MS Teams.

Reviewer 2: same

> - `/msteams-sync show` :- This command is used to show the link of the currently active Mattermost channel. It displays the team name, team ID, channel name and channel ID of the MS Teams channel to which the currently active MM channel is linked.

Reviewer: ```suggestion
- `/msteams-sync show` :- This command is used to show the link of the currently active Mattermost channel. It displays the team name, team ID, channel name and channel ID of the MS Teams to which the currently active MM channel is linked.
```

> - `/msteams-sync show-links` :- This command is used to show all the currently active links and can only be run by system admins. It displays all the links in a tabular form that contains Mattermost team, Mattermost channel, MS Teams team, MS Teams channel.

Reviewer 2: ```suggestion
- `/msteams-sync show-links` :- This command is used to show all the currently active links and can only be run by system admins. It displays all the links that contain the Mattermost team, Mattermost channel, MS Teams team, MS Teams channel.
```
